### PR TITLE
rubocop:auto_correct fixes

### PR DESCRIPTION
**Pull request by traylenator**
#### rubocop corrections
Results of rubocop:auto_correct

### CODE REVIEW

**Comment by traylenator** [comment]
no rubocop CI yet.

**Comment by ekohl** [comment]
Yes, that is something I recently discovered. In the migration from Travis to Github Actions I missed that.

File: spec/classes/dnat4_spec.rb [hunk 1]
```diff
@@ -56,71 +56,71 @@
             owner:          'root',
             group:          'root',
             mode:           '0640',
-            ensure_newline: true,
```
Comment (ekohl) [inline spec/classes/dnat4_spec.rb:59]: FYI: we had some discussion about trailing comma's. I think most people agreed that we should enable them in Rubocop as well. We use them in Puppet so it's inconsistent not to do the same in Ruby. I would prefer that over changing everything in this module.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: spec/classes/dnat4_spec.rb (hunk 1 above)
Comment (traylenator) [reply spec/classes/dnat4_spec.rb:59]: Agreed.

**Pull request closed without merging**